libs/libcommon/src/libcommon/constants.py

```diff
@@ -9,21 +9,21 @@
 QUEUE_TTL_SECONDS = 604_800  # 7 days
 
 PROCESSING_STEP_CONFIG_NAMES_VERSION = 1
-PROCESSING_STEP_CONFIG_PARQUET_VERSION = 3
-PROCESSING_STEP_CONFIG_SIZE_VERSION = 1
-PROCESSING_STEP_CONFIG_INFO_VERSION = 1
-PROCESSING_STEP_DATASET_INFO_VERSION = 1
-PROCESSING_STEP_DATASET_IS_VALID_VERSION = 1
-PROCESSING_STEP_DATASET_PARQUET_VERSION = 1
-PROCESSING_STEP_DATASET_SIZE_VERSION = 1
-PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 1
-PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_STREAMING_VERSION = 1
+PROCESSING_STEP_CONFIG_PARQUET_VERSION = 4
+PROCESSING_STEP_CONFIG_SIZE_VERSION = 2
+PROCESSING_STEP_CONFIG_INFO_VERSION = 2
+PROCESSING_STEP_DATASET_INFO_VERSION = 2
+PROCESSING_STEP_DATASET_IS_VALID_VERSION = 2
+PROCESSING_STEP_DATASET_PARQUET_VERSION = 2
+PROCESSING_STEP_DATASET_SIZE_VERSION = 2
+PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_DATASET_INFO_VERSION = 2
+PROCESSING_STEP_DATASET_SPLIT_NAMES_FROM_STREAMING_VERSION = 2
 PROCESSING_STEP_PARQUET_AND_DATASET_INFO_VERSION = 1
-PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_PARQUET_VERSION = 1
-PROCESSING_STEP_CONFIG_PARQUET_AND_INFO_VERSION = 1
+PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_PARQUET_VERSION = 2
+PROCESSING_STEP_CONFIG_PARQUET_AND_INFO_VERSION = 2
```

Comment:
maybe let's update the image row group size at the same time, to not recompute the parquet files twice

Reply:
hmmm, anyway the version upgrade k8s job is not enabled for now, so, it's not critical, I think. Right @AndreaFrancis ?

Reply:
oh ok !

Reply:
Yes, It is disable with action=skip